apps/astarte_appengine_api/test/astarte_appengine_api_web/plug/group_name_decoder_test.exs

```diff
@@ -0,0 +1,30 @@
+defmodule Astarte.AppEngine.APIWeb.Plug.GroupNameDecoderTest do
+  use Astarte.AppEngine.APIWeb.ConnCase
+  use ExUnitProperties
+  alias Astarte.AppEngine.APIWeb.Plug.GroupNameDecoder
+  alias Astarte.AppEngine.API.GroupTestGenerator
+
+  @max_subpath_count 10
+
+  describe "unit tests to Plug.GroupNameDecoder" do
```

Comment:
This description seems redundant, either make it more informative or remove it (in this case, you might add some more info to the `property` name)